src/node.cc

```diff
@@ -2434,6 +2436,23 @@ static void PrintHelp() {
          "Documentation can be found at https://nodejs.org/\n");
 }
 
+static void PrintCompletion() {
+  printf("_node_complete()\n"
+         "{\n"
+         "  local cur_word options\n"
+         "  cur_word=\"${COMP_WORDS[COMP_CWORD]}\"\n"
+         "  options=\"`$1 --help`\"\n"
```

Comment:
How robust is this? That is how much can we change the output of `node --help` before this breaks?

Reply:
Perhaps @addaleax suggestion about https://github.com/nodejs/node/pull/22490 (which I've not looked into yet) might help making this more robust. I'll take a closer look next week.